### ISSUE TITLE
'Insert snippet' icon displayed for general function documentation

### ISSUE THREAD

**Issue by mjainQA** [Komodo/KomodoEdit#2506]
### Short Summary

Clicking on 'Insert Snippet' is not inserting any snippet of selected documentation. These functions are higher level for e.g. Node.js -> Array. User needs to browse to specific function such as Node.js -> Array -> push to insert code snippet 'push(item1, item2, ...)' 

The icon should not be available for higher level function.

### Steps to Reproduce

1. Go to Commando -> Documentation
2. Select Node.js -> Array
3. Click on 'insert snippet' icon

### Platform Information

*Komodo Edit or IDE?* IDE
*Komodo Version?* 10.2.2 RC2
*Operating System (and version)?* macOS

### Additional Information
![image](https://cloud.githubusercontent.com/assets/26605593/25248200/30687a08-25c2-11e7-8435-c9652299e6e1.png)

**Comment by Naatan** [comment]
Why not? You can use this in your code all the same, I don't agree that the insert snippet button has no place here.

**Comment by cgchoffman** [comment]
What is it supposed to insert? `[]`?  If it is supposed to insert something then it's not working.

**Comment by Naatan** [comment]
No, it's supposed to insert `Array`, which can be used like `Array.isArray()`. Array is itself an object.
